ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixFactory.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.clustermap;
+
+import org.apache.helix.HelixManager;
+import org.apache.helix.HelixManagerFactory;
+import org.apache.helix.InstanceType;
+
+
+/**
+ * A factory class to construct and get a reference to a {@link HelixManager}
+ */
+public class HelixFactory {
```

Comment:
Prefer it this way as there is a `HelixManagerFactory` in Helix and want to avoid confusion.